frames/src/main/java/com/checkout/frames/screen/billingformdetails/models/BillingForm.kt

```diff
@@ -0,0 +1,13 @@
+package com.checkout.frames.screen.billingformdetails.models
+
+import com.checkout.base.model.Country
+import com.checkout.tokenization.model.Address
+import com.checkout.tokenization.model.Phone
+import kotlinx.coroutines.flow.MutableStateFlow
+
+internal data class BillingForm(
```

Comment:
`BillingForm` is more UI-related name than data.
As option could be `BillingAddress`.

Reply:
Yes, true. I have tried to align it with IOS. However, BillingAddress sounds good to me as well.